oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/Layer.java

```diff
@@ -126,7 +126,12 @@ public enum Layer {
     /**
      * Cache conjectured by client side plugin(eg. skywalking-java -> JedisPlugin LettucePlugin)
      */
-    VIRTUAL_CACHE(19, false);
+    VIRTUAL_CACHE(19, false),
+
+    /**
+     * PostgreSQL is an advanced, enterprise-class, and open-source relational database system.
+     */
+    POSTGRESQL(20, true);
```

Comment:
@Fine0830 Please add this layer available to the menu as root, and put PostgreSQL as a sub-menu in the Database side by side with MySQL, name `PostgreSQL`